Tasks/NuGetCommand/nugetrestore.ts

```diff
@@ -158,7 +157,11 @@ export async function run(nuGetPath: string): Promise<void> {
         // Setting creds in the temp NuGet.config if needed
         await nuGetConfigHelper.setAuthForSourcesInTempNuGetConfigAsync();
 
-        let configFile = nuGetConfigHelper.tempNugetConfigPath;
+        // Use config file if:
+        //     - User selected "Select feeds" option
+        //     - User selected "NuGet.config" option and the nuGetConfig input has a value
+        let useConfigFile: boolean = selectOrConfig === "select" || (selectOrConfig === "config" && !!nuGetConfigPath);
```

Comment:
When select feed is selected, in line#150, tempNugetConfigPath is set to nugetConfigPath
When it is config, line#104, nugetConfigPath is set to undefined and created in NuGetConfigHelper2

Does it mean we can use nugetConfigPath?